src/main/java/dan200/computercraft/client/render/text/DirectFixedWidthFontRenderer.java

```diff
@@ -165,19 +165,24 @@ public static void drawTerminalBackground(
         }
     }
 
-    public static void drawTerminalWithoutCursor(
+    public static int drawTerminalWithoutCursor(
         @Nonnull ByteBuffer buffer, float x, float y, @Nonnull Terminal terminal, boolean greyscale,
         float topMarginSize, float bottomMarginSize, float leftMarginSize, float rightMarginSize
     )
     {
-        drawTerminalForeground(
+        drawTerminalBackground(
             buffer, x, y, terminal, greyscale,
             topMarginSize, bottomMarginSize, leftMarginSize, rightMarginSize
         );
-        drawTerminalBackground(
+
+        int pivot = buffer.position();
```

Comment:
Instead of returning the pivot, I'd honestly just call the individual `drawTerminalForeground`/`drawTerminalBackground` methods in `TileEntityMonitorRenderer`.

Reply:
Done :+1: